README.md

```diff
@@ -1,14 +1,144 @@
+What you will need:
 
-### Frontend
+- A free Firebase account
+- Your favorite command-line interface (e.g. Terminal or zsh on a Mac)
+- Optional: accounts with Stripe, Twillio
+- A little fairy dust
+
+---
+<!-- section 1 -->
+
+## Part One: Firebase Project Setup
+
+### Step 1: Create New Firebase Project
+
+1. Go to https://console.firebase.google.com/u/2/.
+
+2. Click _Add project_.
+  * Fill out the name field with something like `example-project`
+  * Click _Continue_
+
+
+3. Optional: enable Google Analytics.
+
+  * Click _Continue_
+  * Configure Google Analytics
+  * Click _Create project_
+  * Wait for project to be created
+  * Click _Continue_
+
+This part of the setup is complete!
+
+
+### Step 2: Configure Firebase Project Settings
+
+1. Go to https://console.firebase.google.com/ and find the `Example Project` you chose in step 1.
+
+2. From _Project Overview_, hover over the gear icon and click _Project Settings_.
+
+ * In the 'General' tab:
+
+    * Set the public-facing name.
+   * In the ['Users & Permissions'](https://console.firebase.google.com/u/2/project/`example-project`/settings/iam) tab:
+
+    * Add any other users who will need "Owner" level access to this project (e.g. a backup owner who can change things if needed, deployments).
+    * Add the users who will need "Editor" level access to this project.
+
+3. Optionally, upgrade your account. Click the gear icon next to _Project Overview_ again, and then select ['Usage and billing'](https://console.firebase.google.com/u/2/project/`example-project`/usage), then select the Blaze plan.
+
+### Step 3: Add a New Web App to the Firebase Project
+
+1. Click on _Project Overview_, then again on [_Project Settings_](https://console.firebase.google.com/u/2/project/`example-project`/settings/general).
+
+ * In the 'General' tab:
+
+    * Go to the 'Your Apps' section at the bottom.
+    * Click on the `</>` button to create a new web app
+    * Add firebase to your web app
+    * Choose an app nickname (eg. `example-project`)
+    * Tick 'Also set up Firebase Hosting for this app', leaving the dropdown that appears on the default setting
+    * Click _Register app_
+    * Wait for the _loading_ spinner to finish
+    * Click _Next_ to skip past this screen
+    * Click _Next_ to skip past this screen as well
+    * Finally, you can click _Continue to the console_ to skip past this last screen; internal deployments are handled automatically through our CircleCI workflow, so we don't need to deploy manually
+
+This part of the setup is now complete!
+
+
+### Step 4: Set up Firebase Hosting
+
+1. From the Firebase console, within the appropriate project, click on ['Hosting'](https://console.firebase.google.com/u/2/project/`example-project`/hosting) on the left hand menu.
+
+2. Click on _Get started_.
+
+ * You can skip through these next screens by clicking _Next_
+  * Click _Continue to the console_
+
+2. OPTIONAL Add a custom domain.
```

Comment:
I believe we should keep same style for "optional" marks. Suggesting the same way as you did it earlier:
```suggestion
2. Optional: Add a custom domain.
```